infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/RemoveNodeWorkflow.java

```diff
@@ -0,0 +1,57 @@
+package org.corfudb.infrastructure.orchestrator;
+
+import lombok.Getter;
+import org.corfudb.protocols.wireprotocol.orchestrator.RemoveNodeRequest;
+import org.corfudb.protocols.wireprotocol.orchestrator.Request;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.Layout;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import static org.corfudb.format.Types.OrchestratorRequestType.REMOVE_NODE;
+
+/**
+ * @author Maithem
+ */
+public class RemoveNodeWorkflow extends Workflow {
```

Comment:
A short file level comment would do for this one.

Reply:
Done.